docs/support.md

```diff
@@ -1,3 +1,7 @@
+<!--meta
+Where can I find help, if I have problems using MithrilJS?
```

Comment:
Could you recast this into a statement or descriptive fragment instead of a question? Maybe something like this, although you can go with something else if you'd prefer:

```suggestion
Where to find help with problems related to MithrilJS
```